pkgs/servers/monitoring/uptime-kuma/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Is that required for the update script? Can't we supply that via a argument?

Reply:
Yes it is required, if I delete .npmrc I get this kind of errors : request to https://registry.npmjs.org/chart.js failed: cache mode is 'only-if-cached' but no cached response is available.
We may be able to supply it via argument but I have no idea how to achieve that.

Reply:
> We may be able to supply it via argument but I have no idea how to achieve that.

We need a custom update script anyway so we should supply it there.

Reply:
@SuperSandro2000 I've added an update script. Sorry I wasn't clear earlier, the .npmrc file is required for building the package, not for the update script.